.gitignore

```diff
@@ -1,3 +1,4 @@
 /taskcluster-worker
 /.vagrant
 coverage.out
+/.bash_history
```

Comment:
It's from when we run `docker-test.sh` I touch and mount it in locally.. So it remembers commands inside the docker environment... Just a neat trick..